### PR TITLE
Feature/dosa 73 create basic html calculator

### PR DESCRIPTION
**Pull request by oliskertch**
Complete

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,34 +1,183 @@
...
+  calculationDiv.textContent = inputArray.join("");
+
+  //clear the input string
+  inputString = "";
+
+}
+
+function runningTotal() {
+
+  const outputDiv = document.getElementById("outputScreen");
```
Comment (jmgcrespo) [inline index.js:135]: I feels a bit that your are mixing two different concerns in this function, calculating the total and displaying the total. Did you consider maybe having a function that displays the total which will call runningTotal?

File: index.html [hunk 2]
```diff
@@ -1,14 +1,121 @@
...
+
 <body>
     <h1>Calculator</h1>
-    
+
     <script src="index.js"></script>
+
+    <div id="case" class="calculator_case">
+
+        <!-- <script>
```
Comment (jmgcrespo) [inline index.html]: you should remove commented code

File: index.js (hunk 1 above)
Comment (oliskertch) [reply index.js:135]: I've taken the lines out which display the total and put them in the functions which do call runningTotal (now renamed calculateRunningTotal): operatorButton and equalsButton. I think this achieves what you suggest.

File: index.html (hunk 2 above)
Comment (oliskertch) [reply index.html]: Done

**Pull request merged**